sap/xssec/jwt_audience_validator.py

```diff
@@ -0,0 +1,112 @@
+class JwtAudienceValidator(object):
+    '''
+     Validates if the jwt access token is intended for the OAuth2 client of this
+     application. The aud (audience) claim identifies the recipients the JWT is
+    issued for.
+
+    Validates whether there is one audience that matches one of the configured
+    OAuth2 client ids.
+    '''
+
+    DOT = "."
+
+    def __init__(self, clientid):
+        self._clientid = clientid
+        self._trusted_clientids = set()
+        self.trusted_clientids = clientid
```

Comment:
@robertofalk Just found this PR. Not sure if line 16 should be there?

Reply:
HI Michael ,
   We will need to automatically add the client_id passed to  the jwt_audience_validator to the trusted client ids list.
Here i am calling the setter self.trusted_clientids = clientid, inside the setter for trusted_clientids, I perform the check 
 if clientid:
            self._trusted_clientids.add(clientid)

and then add it .